templates/components/itilobject/user_picture.html.twig

```diff
@@ -1,4 +1,4 @@
-{% set user_picture  = User__getPicture(users_id) %}
+{% set user_picture  = User__getPicture(users_id|number_format) %}
```

Comment:
I do not understand this one.

Reply:
`users_id` is (was?) not always of type `int`; so cast was required (I did not find where the wrong value was set).

Reply:
What I do not understand is that the `User__getBgColor` and `User__getLinkUrl` had similar signatures and bodies, so if you have to cast `users_id` in `User__getPicture()` call, you should also cast it in other methods. Maybe the best way to handle that is to encapsulate this whole template in a `{% if users_id|number_format > 0 %}...{% endif %}`.

Reply:
Well, I've fixed the cases I've seen; I'm pretty sure several other methods returns a void while a string or null is expected. I'll take a look.

Reply:
Sounds like the wrong return type was alone finally. As fro the users_id, I've just casted it; the (if won't set the value as it would do on a php script - so that would not solve the initial issue).